src/common/time/utils.tsx

```diff
@@ -1,30 +1,33 @@
-import moment from 'moment';
+import { format, parse } from 'date-fns';
 
 import { BACKEND_DATE_FORMAT } from './TimeConstants';
 
 /**
- * Return new instance of moment, same with moment()
- * Use this util function to keep all moment import in single place.
+ * Return new instance of Date, same with new Date()
+ * Use this util function to keep all date-fns import in single place.
  * @param inp
- * @param format
- * @param strict
+ * @param formatStr
  */
-export const newMoment = (
-  inp?: moment.MomentInput,
-  format?: moment.MomentFormatSpecification,
-  strict?: boolean
-) => moment(inp, format, strict);
+export const newDate = (
+  inp?: string | number | Date | null,
+  formatStr?: string
+) => {
+  if (formatStr) {
+    return parse(inp as string, formatStr, new Date());
+  }
+  return inp ? new Date(inp) : new Date();
+};
```

Comment:
The `formatStr` parameter is unused and could be removed as dead code:
```suggestion
export const newDate = (inp?: string | number | Date | null) =>
  inp ? new Date(inp) : new Date();
```
+removing the `parse` import as unnecessary.